### ISSUE TITLE
IOS app crashing UIColor outside of expected range

### ISSUE THREAD

**Issue by dillonstreator** [davecoffin/nativescript-modal-datetimepicker#16]
### Which platform(s) does your issue occur on?
- iOS

### Please, tell us how to recreate the issue in as much detail as possible. 
Install the plugin and run on ios.

### Error thrown inside the debugger
```
[Graphics] UIColor created with component values far outside the expected range, Set a breakpoint on UIColorBreakForOutOfRangeColorComponents to debug. This message will only be logged once.
```

### Findings
Every parameter passed to UIColor must be between 0 and 1 and not 0 to 255

### Is there any code involved?
I was able to resolve the issue by mapping the previous values between 0 and 255 to between 0 and 1 but am unable to make the pull request due to permissions.
The fix that worked for me was changing the following in src/modal-datetimepicker.ios.ts
Line 239
```
UIColor.colorWithRedGreenBlueAlpha(0, 153, 255, 1),
```
to
```
UIColor.colorWithRedGreenBlueAlpha(0, .6, 1, 1),
```

**Comment by marcelomiranda90** [comment]
Same problem here.

**Comment by shiv19** [comment]
Proposed changes made and published as `nativescript-modal-datetimepicker@1.1.4`
